### PR TITLE
feat(ccls-1819): implement hrefs for actions and notifications on hom…

### PR DESCRIPTION
**Pull request by keith-h**
…epage

### CODE REVIEW

File: src/main/resources/templates/home.html [hunk 1]
```diff
@@ -27,10 +27,10 @@ <h2 class="govuk-heading-m">
                 </div>
                 <ul class="govuk-list">
                     <li>
-                        <a th:text="${actionsMsg}" href="/"></a>
+                        <a th:text="${actionsMsg}" href="/notifications?notificationType=A"></a>
```
Comment (PhilDigitalJustice) [inline src/main/resources/templates/home.html:30]: snake_case params

File: src/main/resources/templates/home.html [hunk 2]
```diff
@@ -27,10 +27,10 @@ <h2 class="govuk-heading-m">
...
                     </li>
                     <li>
-                        <a th:text="${notificationsMsg}" href="/"></a>
+                        <a th:text="${notificationsMsg}" href="/notifications?notificationType=N"></a>
```
Comment (PhilDigitalJustice) [inline src/main/resources/templates/home.html:33]: snake_case params

File: src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java [hunk 3]
```diff
@@ -60,21 +60,18 @@ public String getNotifications(
       @RequestParam(value = "page", defaultValue = "0") int page,
       @RequestParam(value = "size", defaultValue = "10") int size,
       @RequestParam(required = false) String sort,
+      @RequestParam(value = "notificationType", required = false) String notificationType,
```
Comment (PhilDigitalJustice) [inline src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java:63]: snake_case params

Comment (PhilDigitalJustice) [reply src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java:63]: value = "notification_type"

**Review by PhilDigitalJustice — CHANGES_REQUESTED** [review]
One thing to change

File: src/main/resources/templates/home.html (hunk 2 above)
Comment (keith-h) [reply src/main/resources/templates/home.html:33]: on it

File: src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java (hunk 3 above)
Comment (keith-h) [reply src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java:63]: on it

File: src/main/resources/templates/home.html (hunk 1 above)
Comment (keith-h) [reply src/main/resources/templates/home.html]: done it

File: src/main/resources/templates/home.html (hunk 2 above)
Comment (keith-h) [reply src/main/resources/templates/home.html]: done it

File: src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java (hunk 3 above)
Comment (keith-h) [reply src/main/java/uk/gov/laa/ccms/caab/controller/ActionsAndNotificationsController.java]: done it

**Review by PhilDigitalJustice — APPROVED** [review]
LGTM

**Pull request merged**